wiki/Performance_troubleshooting/en.md

```diff
@@ -10,6 +10,7 @@ Unfortunately, due to the scope of hardware which osu! runs on, it's very hard t
 - Is your frame limiter set to `Unlimited`? Set it to `Optimal` and see if it helps. See [its section below](#unlimited) for more information on why `Unlimited` is usually not the best choice.
 - Are you running windowed? Switch to fullscreen.
 - Have you updated your drivers (especially GPU)? Try doing so and see if things improve. Alternatively, if you recently updated them, try rolling back.
+- Are you using a hybrid graphics setup (where the dedicated GPU renders the game and the integrated GPU displays the screen)? If so, try running the game on either only integrated GPU or only dedicated GPU (by using a mux switch if available, or by using an external display).
```

Comment:
I dunno about this advice. The only case where this causes issues is if the user is running stupid high frame rates. Also, to make this understandable I'd just call it out for what it is – refer to it using the "optimus" branding because the only time this even comes up is with optimus chips. Then you don't need to try and explain to the user the rendering pipeline.

Reply:
Then at least, I think this article should mention something about optimus, and leave it there for people to search about it themselves, rather than having no info.
I'd also like to have a part about this on official wiki page, since most of the time someone ask for tech support, I literally have to search for some of your tweets (luckily I bookmarked them) just to tell them about this problem.

Reply:
Again, I'm not against that. I'm saying to use the "optimus" term instead of "hybrid graphics setup (where the dedicated GPU renders the game ...)". A very small subset of users will understand the second. Most will choose to not read the sentence.

Reply:
a passerby comment: if this is still an issue, you can refer to "Optimus" in the body of the article and add a footnote that would mention that it is also valid for other hybrid setups (and explain what they are)